cub/device/dispatch/dispatch_spmv_orig.cuh

```diff
@@ -664,12 +664,12 @@ struct DispatchSpmv
 
             // Get grid dimensions
             dim3 spmv_grid_size(
-                CUB_MIN(num_merge_tiles, max_dim_x),
+                CUB_MIN(num_merge_tiles, static_cast<unsigned int>(max_dim_x)),
```

Comment:
These were fixed earlier by b2e64cf0fb4ea7ace6c86ca6765ca7c1087ef82e. I'll drop this while rebasing.